src/platform.rs

```diff
@@ -762,7 +762,7 @@ mod tests {
 
     #[test]
     fn test_get_set_xsave_state() {
-        let mut capability_features: WHV_CAPABILITY_FEATURES;
+        let mut capability_features: WHV_CAPABILITY_FEATURES = Default::default();
         capability_features.AsUINT64 = 0;
```

Comment:
The next assignment would then become redundant but we still have to keep it, otherwise we get a different warning:
warning: value assigned to `capability_features` is never read

I'd say we could merge this.

Reply:
There is no such warning in my build... otherwise I wouldn't have sent the PR :)

Reply:
Indeed, you'll get that only if you remove the next assignment (which may seem redundant). What I'm saying is that the commit is fine, thanks for sending it.